src/main/kotlin/racingcar/view/Output.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.view
+
+import racingcar.domain.Car
+
+class Output {
+    fun outputExecutionResult(carList : List<Car>){
+        carList.forEach { car ->
+            print(car.name + " : ")
+            repeat(car.position) {
+                print("-")
+            }
+            println()
+        }
```

Comment:
이런식으로 짧은 코드도 가능합니다.ㅎㅎ
```suggestion
        carList.forEach { car ->
            println(car.name + " : " + "-".repeat(car.position))
        }
```